modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -114,7 +114,7 @@
                       claim_status_type: 'Compensation',
                       claim_dt: '2017-05-02',
                       phase_type: 'Pending',
-                      attention_needed: true,
+                      attention_needed: 'Yes',
```

Comment:
This is how the response from BGS actual looks ('Yes' or 'No'), so reflecting that here